internal/core/config/validation/validator.go

```diff
@@ -28,14 +31,19 @@ func ValidateCustomConfig(conf interface{}) error {
 
 // ValidateStruct uses the `validate` struct tags to do standard validation
 func ValidateStruct(confStruct interface{}) error {
+	en := en.New()
+	uni := ut.New(en, en)
+	trans, _ := uni.GetTranslator("en")
```

Comment:
What's this stuff for?

Reply:
It is used in the error message generated below to make it a lot more user friendly.  You have to do this convolution to make it English since they support tons of languages.

Reply:
Also, why are you ignoring the error being returned here?

Reply:
Because English is always there in our locked dependency version.